pyqtgraph/functions.py

```diff
@@ -386,7 +386,16 @@ def glColor(*args, **kargs):
     c = mkColor(*args, **kargs)
     return (c.red()/255., c.green()/255., c.blue()/255., c.alpha()/255.)
 
-    
+
+def serializeQtData(obj):
+    """
+    Serializes any Qt data type that can be serialized into a QDataStream using the left-shift operator.
+    """
+    a = QtCore.QByteArray(b'')
+    s = QtCore.QDataStream(a, QtCore.QIODevice.WriteOnly)
+    s << obj
+    return bytes(a)
```

Comment:
this is the line generating the segmentation fault.

Reply:
I think you want to return `a.data()`

Reply:
after changing this to return `a.data()` the test suite passed for me 👍

Reply:
Thanks for digging into this.

Reply:
no problem, thanks for the PR and incorporating the fix 👍